src/components/ConcentratedLiquidity/PositionCard/index.tsx

```diff
@@ -72,20 +72,26 @@ const PositionCard: React.FC<PositionCardProps> = (props) => {
                     %{feeAmount / 10 ** 4}
                   </BlackBoxContent>
                 </BlackBox>
-                <BlackBox>
-                  <BlackBoxContent color="color11" fontSize={18} fontWeight={500}>
-                    {!liquidity?.isZero() ? t('common.open') : t('common.closed')}
-                  </BlackBoxContent>
-                </BlackBox>
-                <BlackBox>
+                <BlackBox data-tip data-for={`positionStatus-${tokenId}`}>
                   <BlackBoxContent color="color11" fontSize={18} fontWeight={500}>
-                    {closed
-                      ? t('common.closed')
-                      : outOfRange
-                      ? t('concentratedLiquidity.positionCard.outOfRange')
-                      : t('concentratedLiquidity.positionCard.inRange')}
+                    {!closed ? t('common.open') : t('common.closed')}
                   </BlackBoxContent>
+                  <Tooltip id={`positionStatus-${tokenId}`} effect="solid">
+                    {!closed ? 'Tooltip Open Content' : 'Tooltip Closed Content'}
+                  </Tooltip>
                 </BlackBox>
+                {!closed && (
+                  <BlackBox data-tip data-for={`positionRangeStatus-${tokenId}`}>
+                    <BlackBoxContent color="color11" fontSize={18} fontWeight={500}>
+                      {outOfRange
+                        ? t('concentratedLiquidity.positionCard.outOfRange')
+                        : t('concentratedLiquidity.positionCard.inRange')}
+                    </BlackBoxContent>
+                    <Tooltip id={`positionRangeStatus-${tokenId}`} effect="solid">
+                      {outOfRange ? 'Tooltip OutOfRange Content' : 'Tooltip InRange Content'}
```

Comment:
lets update this tooltip text & also lets put it inside locale files.

for out of range:  `The price of this pool is outside of your selected range. Your position is not currently earning fees.`

for in range: `The price of this pool is within your selected range. Your position is currently earning fees.`